skimage/filters/_gaussian.py

```diff
@@ -83,23 +84,29 @@ def gaussian(image, sigma=1, output=None, mode='nearest', cval=0,
     >>> from skimage.data import astronaut
     >>> image = astronaut()
     >>> filtered_img = gaussian(image, sigma=1, multichannel=True)
-
     """
-
     spatial_dims = guess_spatial_dimensions(image)
     if spatial_dims is None and multichannel is None:
         msg = ("Images with dimensions (M, N, 3) are interpreted as 2D+RGB "
                "by default. Use `multichannel=False` to interpret as "
                "3D image with last dimension of length 3.")
         warn(RuntimeWarning(msg))
         multichannel = True
+
     if np.any(np.asarray(sigma) < 0.0):
         raise ValueError("Sigma values less than zero are not valid")
+
+    # Handle longstanding SciPy bug with NumPy integers passed as sigma
```

Comment:
if there was a bug, can you think of a test that would break without your correction?